dwave/optimization/states.pyx

```diff
@@ -137,6 +138,7 @@ cdef class States:
         self._states.swap(states)
         return move(states)
 
+    @_file_object_arg("rb")  # translate str/bytes file inputs into file objects
```

Comment:
These are redundant for now, but will be used in https://github.com/dwavesystems/dwave-optimization/pull/241